tools/autograd/load_derivatives.py

```diff
@@ -383,5 +383,5 @@ def find_info(declaration):
     for declaration in declarations:
         info = find_info(declaration)
         declaration['derivative'] = info['autograd_fn'] if info else None
-        declaration['non_differentiable_arg_names'] = info['non_differentiable_arg_names'] if info else None
+        declaration['non_differentiable_arg_names'] = info['non_differentiable_arg_names'] if info else []
```

Comment:
Why have 'non_differentiable_arg_names' and not 'differentiable_arg_names' if we're doing all this work already?

Reply:
It's not a bad idea, but would take a lot more code restructuring than is necessary to accomplish schema unification.

For example, there are multiple args_with_derivatives definitions in gen_variable_type that are different.  You'd have to correct this to get something sensible.

Reply:
Ok, maybe we'll do that down the road then.